singer-connectors/tap-mysql/requirements.txt

```diff
@@ -1 +1 @@
-pipelinewise-tap-mysql==1.0.7
+-e git+https://github.com/transferwise/pipelinewise-tap-mysql.git@support-binary-cols#egg=pipelinewise-tap-mysql
```

Comment:
This is only for tests, right? In the final version you can put `1.1.1` here.

Reply:
I'll update this once the singer components are released.